src/components/Code.vue

```diff
@@ -1,12 +1,19 @@
 <template>
   <div class="code">
-    <div class="card" v-if="language === 'html'">
+    <div
+      class="card"
+      v-if="language === 'html'"
+      v-bind:class="{ 'border-danger': wasResponseAnError }">
```

Comment:
You can drop the `v-bind`. `:class` is sufficient.

Reply:
Done